jupyter_server/extension/application.py

```diff
@@ -338,7 +342,8 @@ def launch_instance(cls, argv=None, **kwargs):
         # Get a jupyter server instance.
         serverapp = cls.initialize_server(
             argv=args, 
-            load_other_extensions=cls.load_other_extensions
+            load_other_extensions=cls.load_other_extensions,
+            subcommands = cls.subcommands
```

Comment:
Is there a reason to pass subcommands to the underlying `ServerApp`? I don't think this is needed.